subsys/bluetooth/audio/Kconfig.has

```diff
@@ -15,6 +15,12 @@ menuconfig BT_HAS
 
 if BT_HAS
 
+config BT_HAS_FEATURES_NOTIFIABLE
+	bool "Hearing Aid Features Notifiable Support"
+	help
+	  This option enables support for clients to subscribe for notifications
+	  on the Hearing Aid Features characteristic.
```

Comment:
The HAS API does not allow to change any of the `feature` bits on the fly, as those `features` are initialized from `bt_has_register` function context. Thus I don't think it's needed to mark Notify property bit if the value won't change.

Reply:
I think you are correct - however, we do have an issue open to add this, and it is an API change that could come in the future.

Does it make more sense if I add that too? It shouldn't be a big deal.

Reply:
I think it makes sense to have the API regardless, as you can have other characteristics which as an API for changing, but still notify as optional. In PACS we have available contexts for example.